scripts/local_testnet/README.md

```diff
@@ -1,201 +1,70 @@
-tar xvf geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
-```
+It will build Lighthouse and will take an approximately 12 minutes to complete. Once built, the testing will be started automatically. You will see a list of services running and "Started!" at the end. 
 
-After extraction, copy `geth` and `bootnode` to the `PATH`. A typical directory is `/usr/local/bin`.
+To view all running services:
 
 ```bash
-cd geth-alltools-linux-amd64-1.13.1-3f40e65c
-sudo cp geth bootnode /usr/local/bin
+kurtosis enclave inspect local-testnet
 ```
 
-After that We can remove the downloaded files:
+To view the logs:
 
 ```bash
-cd ~
-rm -r geth-alltools-linux-amd64-1.13.1-3f40e65c geth-alltools-linux-amd64-1.13.1-3f40e65c.tar.gz
+kurtosis service logs local-testnet $SERVICE_NAME
 ```
 
-We are now ready to start a local testnet.
-
-## Starting the testnet
-
-To start a testnet using the predetermined settings:
+where `$SERVICE_NAME` is obtained by inspecting the running services above. For example, to view the logs of the first beacon node, validator client and geth:
 
 ```bash
-cd ~
-cd ./lighthouse/scripts/local_testnet
-./start_local_testnet.sh genesis.json
+kurtosis service logs local-testnet -f cl-1-lighthouse-geth 
+kurtosis service logs local-testnet -f vc-1-geth-lighthouse
+kurtosis service logs local-testnet -f el-1-geth-lighthouse
 ```
 
-This will execute the script and if the testnet setup is successful, you will see "Started!" at the end. 
-
-The testnet starts with a post-merge genesis state. 
-The testnet starts a consensus layer and execution layer boot node along with `BN_COUNT`
-(the number of beacon nodes) each connected to a geth execution client and `VC_COUNT` (the number of validator clients). By default, `BN_COUNT=4`, `VC_COUNT=4`. 
-
-The `start_local_testnet.sh` script takes four options `-v VC_COUNT`, `-d DEBUG_LEVEL`, `-p` to enable builder proposals and `-h` for help. It also takes a mandatory `GENESIS_FILE` for initialising geth's state.
-A sample `genesis.json` is provided in this directory.
-
-The options may be in any order or absent in which case they take the default value specified.
-- VC_COUNT: the number of validator clients to create, default: `BN_COUNT`
-- DEBUG_LEVEL: one of { error, warn, info, debug, trace }, default: `info`
-
-The `ETH1_BLOCK_HASH` environment variable is set to the block_hash of the genesis execution layer block which depends on the contents of `genesis.json`. Users of these scripts need to ensure that the `ETH1_BLOCK_HASH` variable is updated if genesis file is modified.
-
-To view the beacon, validator client and geth logs:
+Kurtosis comes with a Dora explorer which can be opened with:
 
 ```bash
-tail -f ~/.lighthouse/local-testnet/testnet/beacon_node_1.log
-tail -f ~/.lighthouse/local-testnet/testnet/validator_node_1.log
-tail -f ~/.lighthouse/local-testnet/testnet/geth_1.log
+open $(kurtosis port print local-testnet dora http)
```

Comment:
I think it would also be useful to mention the `kurtosis web` interface

Reply:
Add in [e128165](https://github.com/sigp/lighthouse/pull/5896/commits/e1281658e73ff5d4b700e087118e2677f34d9437)